src/components/general/IconButton/styles.less

```diff
@@ -74,6 +77,29 @@
         
     }
 
+    &:focus { 
+        .iconContainer {
+            border: 1px;
+            border-radius: 50%; 
+            border-style: dashed;
+            border-color: var(--color-primary);
+            }
+
+        &.isToggler{
+             .iconContainer{
+                 border: none;
+                 background-color: var(--color-primary-hover-alt1);
+            }
+
+              &.isActive{ 
+                    .iconContainer{
+                        background-color: var(--color-primary);

```

Comment:
Is there a missing `{` here? 🤔

Reply:
Didn´t fint the missing {, but I fixed the indenting so its better to read.